tests/sentry/tasks/test_deletion.py

```diff
@@ -140,6 +146,83 @@ def test_handle_missing_record(self):
         assert not ScheduledDeletion.objects.filter(id=schedule.id).exists()
 
 
+class DeleteOrganizationTest(TestCase):
```

Comment:
There should be some existing organization tests in `tests/sentry/deletions/test_organization.py` should these go with the existing tests?

Reply:
Ahh, I didn't see this before. Good suggestion! I think it does makes sense to move it here.